doc/releases/nautilus.rst

```diff
@@ -216,13 +216,6 @@ Instructions
         "ceph version 14.2.0 (...) nautilus (stable)": 22,
      }
 
-#. Scan for any OSDs deployed with the old ceph-disk tool to ensure
```

Comment:
We should still keep a section about this in the docs, in case the automation fails.